### DISCUSSION TITLE
mcstart working, sudo systemctl start meshcentral.service not working

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#4987] [General]
Ubuntu 18.04 LTS on Digital Ocean.

Can start server manually using the `./mcstart` shortcut but `sudo systemctl start meshcentral.service` doesn't start it.

I had the dreaded nodes and npm version issues but got them both to work using hints from here:

https://meshcentral-community.com/doku.php?id=howto:installation:manual:ubuntu

`node: /usr/bin/node /home/pc-support/.nvm/versions/node/v14.17.0/bin/node /usr/share/man/man1/node.1.gz`

node -v `v14.17.0`
npm -v `9.5.0`

/etc/systemd/system/meshcentral.service

```
[Unit]
Description=MeshCentral Server

[Service]
Type=simple
LimitNOFILE=1000000
ExecStart=/usr/bin/node /home/pc-support/.nvm/versions/node/v14.17.0/bin/node
WorkingDirectory=/home/pc-support
Environment=NODE_ENV=production
User=pc-support
Group=pc-support
Restart=always
# Restart service after 10 seconds if node service crashes
RestartSec=10
# Set port permissions capability
AmbientCapabilities=cap_net_bind_service

[Install]
WantedBy=multi-user.target
```

Why is my server not starting automatically? There are no error messages to give me a clue.

**Comment by silversword411** [comment]
Look at systemd logs, not mesh logs
